tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -1580,8 +1580,71 @@ public void Single_Archive_File_Rolls_Correctly()
 
                 SimpleConfigurator.ConfigureForTargetLogging(ft, LogLevel.Debug);
 
-                // we emit 5 * 250 *(3 x aaa + \n) bytes
-                // so that we should get a full file + 3 archives
+                // we emit 2 * 250 *(aaa + \n) bytes
```

Comment:
Firstly the comment was wrong (it was probably copied from somewhere else), so I fixed it.